app/controllers/admin/questions_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Admin::QuestionsController < ApplicationController
+  def index
+    @bank_questions = Question.for_banks
+    @partner_questions = Question.for_partners
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def create
+    @question = Question.create(question_params)
+    if @question.valid?
+      redirect_to admin_questions_path
+    else
+      flash[:error] = "Failed to create question."
+      render :new
+    end
+  end
+
+  def edit
+    @question = current_question
+  end
+
+  def update
+    @question = current_question
+    @question.update(question_params)
+    if @question.valid?
+      redirect_to admin_questions_path
+    else
+      flash[:error] = "Failed to create question."
```

Comment:
Same suggestion as above to include indication on why it failed.

Reply:
I added text to the error message that says that the "Answer can't be blank." if the answer was blank for creating and updating questions.